katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/JoinRegistry.java

```diff
@@ -0,0 +1,115 @@
+package io.katharsis.jpa.internal.query;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.persistence.criteria.From;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.persistence.criteria.From'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)